.travis.yml

```diff
@@ -34,4 +34,5 @@ cache:
 
 branches:
   only:
-    - rm-census
\ No newline at end of file
+    - master
+    - census-version
```

Comment:
This is going to push branch build images to our registry which will get deployed into our environments, I don't think we want that.

Reply:
Ade is just having a play with travis, he'll revert it